### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/NServiceBus.SqlServer/Configuration/ConnectionPoolValidator.cs [hunk 1]
```diff
@@ -15,15 +15,14 @@ public static ValidationCheckResult Validate(string connectionString)
                 return ValidationCheckResult.Valid();
             }
 
-            if (!keys.ContainsKey("Max Pool Size"))
+            if (keys.ContainsKey("Max Pool Size"))
```
Comment (MarcinHoppe) [inline src/NServiceBus.SqlServer/Configuration/ConnectionPoolValidator.cs]: I'm not sure what is the value of this change is, but I'll be happy to learn.

Comment (SimonCropp) [inline src/NServiceBus.SqlServer/Configuration/ConnectionPoolValidator.cs]: two things. 

 1. the fewer logic operations in code the easier it is to reason about. when reading/reviewing the code and u see a negation that is a mental operation your brain needs to store. when reviewing the next few lines. it is why people often get confused by double negatives.
 2. a single negation `!` is easy to miss and stuff up the logic

so when writing code, if the inversion of the negation is semantically equivalent, i will always avoid the negation.

Comment (MarcinHoppe) [inline src/NServiceBus.SqlServer/Configuration/ConnectionPoolValidator.cs]: I can agree for a single conditional. Already merged.

Comment (SimonCropp) [inline src/NServiceBus.SqlServer/Configuration/ConnectionPoolValidator.cs]: yeah it is not always so clear. there are also other things like reducing nesting and early exits to consider.

**Pull request merged**